service/eventer/github/client.go

```diff
@@ -82,6 +86,78 @@ func (e *Eventer) filterDeploymentsByStatus(deployments []deployment) []deployme
 	return matches
 }
 
+func (e *Eventer) fetchLatestDeploymentEvent(project, environment string) (deployment, error) {
```

Comment:
Heavily copied from `fetchNewDeploymentEvents` and modified according to its use case.